<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaMenuPrincipal</class>
 <widget class="QMainWindow" name="ventanaMenuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1900</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1900</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Men√∫ principal del sistema</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelFondoMenuEditarPerfil">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1900</width>
      <height>831</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>menuEditarPerfilFigma.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="labelRealizarEdicion">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>730</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelVolverInicio">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>0</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewVerPerfiles">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>1861</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelConfiguracion">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
